src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -21,10 +21,41 @@ button.mat-button-wrap-text-with-icon {
   border: solid 1px rgba(0, 0, 0, 0.12);
   margin-top: 12px;
 
-  mat-icon {
+  a {
     position: absolute;
-    left: 5%;
-    top: 20%;
+    left: 20px;
+    top: 5px;
+    img {
+      width: 24px;
+    }
+  }
+
+  .columns-4 & {
+    span {
+      display: inline-block;
+      width: 180px;
+      margin: auto;
+      word-break: break-word;
+      white-space: normal;
+      line-height: 24px;
```

Comment:
Not requesting a change here but just FYI: normally define font relate attributes like the `line-height` should be avoided so we do not overwrite font-related features from the app fonts.

Reply:
Ok